### PR TITLE
Dotnet server

### PR DESCRIPTION
**Pull request by zhonghua-wu**
merge .net code to v2

### CODE REVIEW

**Comment by zhonghua-wu** [comment]
merge code to v2

File: StickersTests/obj/Debug/net6.0/.NETCoreApp,Version=v6.0.AssemblyAttributes.cs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// <autogenerated />
```
Comment (NewFuture) [inline StickersTests/obj/Debug/net6.0/.NETCoreApp,Version=v6.0.AssemblyAttributes.cs:1]: ignore

File: server.net/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (KroosHoo) [inline server.net/.vscode/launch.json:1]: Add these debug generated files to git ignore

File: server.net/Program.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
+using Microsoft.Bot.Builder;
```
Comment (KroosHoo) [inline server.net/Program.cs:1]: rename this file name

**Pull request merged**